### PR TITLE
Update sound-blaster-play3 from 2.00.03 to 2.01.02

### PR DESCRIPTION
**Pull request by suschizu**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [ ] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [ ] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

### CODE REVIEW

**Comment by suschizu** [comment]
```
Error: Failure while executing; `/usr/bin/sudo -E -- /usr/bin/env LOGNAME=travis USER=travis USERNAME=travis /usr/sbin/installer -pkg /usr/local/Caskroom/sound-blaster-play3/2.01.02/Install.pkg -target / -verboseR` exited with 1. Here's the output:
315installer: Cannot install on volume / because it is disabled.
316installer: macOS version 10.15 or later is required.
```

**Comment by miccal** [comment]
This updated version is for macOS 10.15 only:

Screen Shot 2020-07-02 at 06 17 22

The old version is for macOS 10.9 to 10.14:

Screen Shot 2020-07-02 at 06 17 38

Perhaps this Cask would benefit from a [`if MacOS.version` block](https://github.com/Homebrew/homebrew-cask/tree/master/doc/cask_language_reference#conditional-statements).

**Comment by suschizu** [comment]
```
audit for sound-blaster-play3: warning
 - only a single uninstall stanza is allowed
```
just this one left 😬

File: Casks/sound-blaster-play3.rb [hunk 1]
```diff
@@ -14,7 +21,11 @@
                        'com.creative.Creative USB Audio Driver.TruStdPlugin109.pkg',
                        'com.creative.Creative Uninstaller.pkg',
                        'com.creative.Creative Product Registration.pkg',
+                       'com.Creative.Sound-Blaster-Play--3-Control-Panel',
+                       'com.creative.AudioDriver',
+                       'com.creative.Uninstaller',
                      ]
+  uninstall launchctl: 'com.creative.trustudiopc'
```
Comment (miccal) [inline Casks/sound-blaster-play3.rb:28]: This should just be `launchctl: 'com.creative.trustudiopc'`

Comment (suschizu) [reply Casks/sound-blaster-play3.rb]: thank you :)

Comment (miccal) [reply Casks/sound-blaster-play3.rb]: You're welcome!

**Pull request merged**